packages/augur-ui/src/services/browser-mesh.ts

```diff
@@ -105,42 +106,33 @@ export async function createBrowserMesh(
   }
 
   try {
-    const zeroXTimerLabel = 'ZeroX Wasm load duration: ';
-    const newMeshTimerLabel = 'new Mesh() startup duration: ';
-
-    zeroX.client.events.emit(SubscriptionEventName.ZeroXStatusStarting, {});
-    console.time(zeroXTimerLabel);
     await loadMeshStreamingWithURLAsync('zerox.wasm');
 
     const meshConfig = createBrowserMeshConfig(
       config.ethereum.http,
-      web3Provider,
+      undefined,
```

Comment:
Ah yes, this is the killer as to why this needs to be in the main thread. In order for this to be feasible we would need to comlink proxy all the eth calls back for this too.

Reply:
I will see if I can do that with comlink.